.github/issue_template/03_mybuildisbroken.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 😵 My Build is Broken
```

Comment:
Oh man, this is such a Pandora's box!

I think we should be super clear about our support policy for these ("maybe if we get around to it"). But I don't want to be a jerk about it.

Reply:
I added a note to the template saying something to that affect. Feel free to correct it when I push my latest change.